.cirrus.yml

```diff
@@ -60,6 +61,25 @@ task:
         mkdir javadoc_tmp
         ./flutter/tools/gen_javadoc.py --out-dir javadoc_tmp
 
+build_windows_task:
+  azure_container_instance:
```

Comment:
Why don't you just use `windows_container` on a Community Cluster? Should have the same effect.

Reply:
Because it's never scheduled in 1 hour: https://cirrus-ci.com/task/5789415858241536...

If I'm using my own azure instances, I can put more CPUs and other resources on the task, right?

Reply:
You'll need to specify `os_version: 1803` for `windows_container`. Windows Containers can only be executed on the same host OS.